requirements.txt

```diff
@@ -1,4 +1,3 @@
-beautifulsoup4==4.4.0
 lxml==3.4.4
```

Comment:
lxml is now on version 3.7.3 and the earliest version they list on PyPI is 3.6.4. Maybe we should bump this up while we're here and check everything is working.